packages/react-components/src/components/SendBox.tsx

```diff
@@ -39,6 +40,59 @@ export interface SendBoxStylesProps extends BaseCustomStyles {
   systemMessage?: IStyle;
 }
 
+/**
+ * Meta Data containing information about the uploaded file.
+ * @beta
+ */
+export interface FileMetadata {
+  /**
+   * File name to be displayed.
+   */
+  name: string;
+  /**
+   * Extension is used for rendering the file icon.
+   * An unknown extension will be rendered as a generic icon.
+   * Example: `jpeg`
+   */
+  extension: string;
+  /**
+   * Download URL for the file.
+   */
+  url: string;
+}
+
+/**
+ * Contains the state attibutes of a file upload like name, progress etc.
```

Comment:
View my comment above about changing this prop name here.
Which means that the documentation should also change.

Something like...
`Attributes required for SendBox to show file uploads as file cards`